app/src/test/java/org/mozilla/focus/utils/UrlUtilsTest.java

```diff
@@ -16,104 +16,104 @@
 public class UrlUtilsTest {
     @Test
     public void isValidSearchQueryUrl() {
-        assertTrue(UrlUtils.isValidSearchQueryUrl("https://example.com/search/?q=%s"));
-        assertTrue(UrlUtils.isValidSearchQueryUrl("http://example.com/search/?q=%s"));
-        assertTrue(UrlUtils.isValidSearchQueryUrl("http-test-site.com/search/?q=%s"));
-        assertFalse(UrlUtils.isValidSearchQueryUrl("httpss://example.com/search/?q=%s"));
+        assertTrue(UrlUtils.INSTANCE.isValidSearchQueryUrl("https://example.com/search/?q=%s"));
```

Comment:
Not worth changing now given all that's piled on top of this but I don't love this churn. `INSTANCE` shouldn't be necessary with `@JvmStatic`, right?

Reply:
Whoops.